pkgs/development/libraries/openmpi/default.nix

```diff
@@ -23,6 +26,13 @@ let
     name = "${cudatoolkit.name}-unsplit";
     paths = [ cudatoolkit.out cudatoolkit.lib ];
   };
```

Comment:
This should be addressed in a separate commit, but I'll just note that we need to eliminate `cudatoolkit` from here, and somehow reproduce this with `cudaPackages.cuda*`. Otherwise the runtime closure blows up. I also wonder if the `symlinkJoin` is absolutely necessary (also bad for the closure)